server/controllers/BookController.js

```diff
@@ -42,6 +42,33 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+* pagination
+* @name pagination
+* @param {Object} request
+* @param {Object} response
+* @returns {JSON} object
+*/
+  static async pagination(request, response) {
+    const { page, limit } = request.query;
+    let pageLimit = limit;
+    let offset = 0;
+    pageLimit = +pageLimit ? pageLimit : pageLimit = 10;
```

Comment:
I don't understand this line

Reply:
to have a default limit if none is given

Reply:
you have two equal signs and what is the plus sign for?

Reply:
i will request you study the code well,